xds/src/main/java/io/grpc/xds/XdsLoadStatsManager.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2019 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.xds;
+
+import io.envoyproxy.envoy.api.v2.core.Locality;
+import io.grpc.LoadBalancer.PickResult;
+import javax.annotation.concurrent.NotThreadSafe;
+
+/**
+ * An {@link XdsLoadStatsManager} is in charge of recording client side load stats, collecting
+ * backend cost metrics and sending load reports to the remote balancer.
+ */
+@NotThreadSafe
+interface XdsLoadStatsManager {
+
+  /**
+   * Establishes load reporting communication and negotiates with the remote balancer to report load
+   * stats periodically.
+   *
+   * <p>This method should be the first method to be called in the lifecycle of
+   * {@link XdsLoadStatsManager} and should only be called once.
+   *
+   * <p>This method is not thread-safe and should be called from the
+   */
+  void startLoadReporting();
+
+  /** Terminates load reporting.
+   * 
+   * <p>No method in {@link XdsLoadStatsManager} should be called after calling this method.
```

Comment:
This should also be called from sync context.

Reply:
Done.